l10n/en-US/viewer.ftl

```diff
@@ -349,6 +349,7 @@ pdfjs-editor-ink-opacity-input = Opacity
 pdfjs-editor-stamp-add-image-button =
     .title = Add image
 pdfjs-editor-stamp-add-image-button-label = Add image
+pdfjs-editor-highlight-thickness-input = Thickness
```

Comment:
Not sure how to pair thickness and higlighting. 

In Nightly I only see Highlight which applies on text. Is "free highlight" the same?

Reply:
Last week we added the possibility to make a free highlight: it's useful to highlight something without a text behind.
The thickness parameter will only apply to the free highlight.